geth/node/manager.go

```diff
@@ -31,86 +34,90 @@ var (
 	ErrRPCClient                   = errors.New("failed to init RPC client")
 )
 
+const (
```

Comment:
Take a look at https://github.com/golang/go/wiki/Iota

Reply:
I think this is not good practice to use iota: any declaration order changes and code could be broken.

Reply:
Do you depend on the order here? I thought each state is referenced by a name and it does not matter what value it has.

Not a big deal here not using iota as it's only 3 states.

Reply:
Ok, but how `iota` makes this code better? I think both codes with and without iota are acceptable. Why we need prefer one solution?
I'll try to describe better: if we have some practice and this practice are common for all devs we should write this practice in some doc like "Code Review practices". But if some practice isn't common why we should check it in code review?

Reply:
In my opinion, it makes this code easier to read and maintain. Also, it clearly states what is done here: definition of incrementing numbers. I can guess here that these values are not really important (and I would probably be right) but `iota` makes it 100% clear.

We don't have such a guideline yet. Generally, we based on https://golang.org/doc/effective_go.html and it states:
> In Go, enumerated constants are created using the iota enumerator.

But it's definitely a good idea to collect such things in one document.

Reply:
@tiabc @adambabik I`ve made >80% test coverage for node manager also iota has been added. But i still think that it`s impossible to do good unit test coverage without mocks -> interfaces.